app/components/ui/switch.tsx

```diff
@@ -0,0 +1,29 @@
+"use client"
```

Comment:
remove use client from everywhere in project . this is the feature of next js (other framework), not remix

Reply:
remove it later, i dont want to hold this pr for too long